inception-imls-lapps/src/test/java/de/tudarmstadt/ukp/inception/recommendation/imls/lapps/LappsRecommenderIntegrationTest.java

```diff
@@ -120,7 +120,7 @@ private Dispatcher buildDispatcher()
             public MockResponse dispatch(RecordedRequest request) {
                 try {
                     String url = request.getPath();
-                    String body = request.getBody().readUtf8();
+                    //String body = request.getBody().readUtf8();
```

Comment:
Don't just comment out code, delete the line if it is not necessary. Do this with all.

Reply:
useless assignment deleted

Reply:
deleted this commented out useless assignment